src/Layout/Layout.tsx

```diff
@@ -0,0 +1,35 @@
+
+import { Link, Outlet } from "react-router-dom";
+import { Strings } from "../resource/Strings";
+
+const Layout = () => {
```

Comment:
Ideally path should be `Layout/Page/index.tsx`

Reply:
I changed the name here